### PR TITLE
Added clarification to README

### PR DESCRIPTION
**Pull request by StandingPadAnimations**
Added clarification that the new rule only applies to assets (applying to all of MCprep is a violation of GPL as I later realized)

### CODE REVIEW

**Comment by StandingPadAnimations** [comment]
Before we merge this, I'll email the FSF regarding the topic of JSON models, since (unless I'm mistaken and the JSON models aren't from the vanilla resource pack) JSON models could actually count as code under GPL

**Comment by StandingPadAnimations** [comment]
Yeah, looks like whatever we do, it's going to be a grey area:
```
Thank you for contacting us and for your willingness to abide by the GPL. Let me start by explaining that the Free Software Foundation (FSF) is working to secure freedom for computer users by promoting the development and use of free software. Please note that we at the FSF do not and cannot help developing proprietary
software, even indirectly. Not only we find it morally wrong, we are also a small organization with limited resources that cannot afford to lose sight of our goal, which is free software. Also, we do not provide legal advice.

As far as I understand your project is an effort to improve rendering animations from a nonfree game. The game is released under a nonfree EULA and it's copyright holder attempts to control modding efforts and reuse of the game's data files by a policy, which can be changed at any time. Your project encourages users to use the nonfree game and results in them being unable to exercise the freedom.

Projects based on proprietary software, such as yours, knowingly live in a gray area. These projects are often one update of a proprietary EULA, or one cease-and-desist letter away from oblivion. So even your best efforts, which are absolutely commendable, to do the right thing in regards with the GPL are overshadowed by this unfortunate state of affairs.

Even though your project is released under the GPL and even though it is an improvement to the free software program Blender, the freedom that the GPL grants cannot be reconciled with the restrictions on the game files imposed by the copyright holder to the game.
```

The key part is this:
> Even though your project is released under the GPL and even though it is an improvement to the free software program Blender, the **freedom that the GPL grants cannot be reconciled with the restrictions on the game files imposed by the copyright holder to the game.**

I think for now, we'll just add the clarification, and then work towards not distributing Minecraft assets (instead asking users to provide a copy of the Minecraft JAR file for assets). I have sent a follow-up regarding requiring users to supply the Minecraft assets themselves, but I think it's fine if we merge this for now.

**Comment by TheDuckCow** [comment]
Let me know if we're still waiting for some kind of response here

**Review by TheDuckCow — APPROVED** [review]

**Pull request merged**
